core/backend/src/ECSchemaXmlContext.ts

```diff
@@ -17,31 +17,52 @@ export type SchemaKey = IModelJsNative.ECSchemaXmlContext.SchemaKey;
 /** @internal */
 export type SchemaMatchType = IModelJsNative.ECSchemaXmlContext.SchemaMatchType;
 
-/** @internal */
+/** Context for deserializing ECSchemas
+ * @beta
+ */
 export class ECSchemaXmlContext {
   private _nativeContext: IModelJsNative.ECSchemaXmlContext | undefined;
 
   constructor() {
     this._nativeContext = new IModelHost.platform.ECSchemaXmlContext();
   }
 
+  /** @internal */
   public get nativeContext(): IModelJsNative.ECSchemaXmlContext {
     assert(undefined !== this._nativeContext);
     return this._nativeContext;
   }
 
+  /**
+   * Adds a file path that should be used to search for a matching schema name
+   * @param searchPath Path to the directory where schemas can be found
+   */
   public addSchemaPath(searchPath: string): void {
     this.nativeContext.addSchemaPath(searchPath);
   }
 
+  /**
+   * Set the last locater to be used when trying to find a schema
+   * @param locater Locater that should be used as the last locater when trying to find a schema
+   * @internal
+   */
   public setSchemaLocater(locater: IModelJsNative.ECSchemaXmlContext.SchemaLocaterCallback): void {
     this.nativeContext.setSchemaLocater(locater);
   }
 
+  /**
+   * Adds a schema locater as first to the current context
```

Comment:
I assume "first" means "searched for schema matches before other locaters". Can you rephrase the docs so I don't have to make assumptions?